pages/common/carbon-now.md

```diff
@@ -0,0 +1,36 @@
+# carbon-now
+
+> Create beautiful images of code.
+> More at <https://github.com/mixn/carbon-now-cli> and <https://carbon.now.sh>.
```

Comment:
I'm unsure how we go with this. Do we have any examples of multiple `More Information` links? Maybe we should remove one of the links. I'll wait for other comments about this. 
Either way, even if we decide to go with both links, it should be as below: 
```suggestion
> More information: <https://github.com/mixn/carbon-now-cli> and <https://carbon.now.sh>.
```

Reply:
I prefer only 1 link, the website doesn't have information about the CLI app, so I vote for this: `https://github.com/mixn/carbon-now-cli`

Reply:
Ok, I agree with @navarroaxel here.

Reply:
I  have removed the `https://cabon.now.sh` link